backend/backend/graphene/queries/syncing.py

```diff
@@ -162,6 +166,26 @@ def resolve_railway_projects(root, info, credential_id):
     except Exception as ex:
         raise GraphQLError(f"Error listing Railway environments: {str(ex)}")
 
+def resolve_vercel_projects(root, info, credential_id):
+    """Resolver for listing Vercel projects."""
+    try:
+        if not test_vercel_creds(credential_id):
+            raise GraphQLError(
+                "Could not authenticate with Vercel. Please check that your credentials are valid"
+            )
+        
+        projects = list_vercel_projects(credential_id)
+        return projects
+    except Exception as ex:
+        raise GraphQLError(f"Error listing Vercel projects: {str(ex)}")
+
+def resolve_test_vercel_creds(root, info, credential_id):
+    """Resolver for testing Vercel credentials."""
+    try:
+        valid = test_vercel_creds(credential_id)
+        return valid
+    except Exception as ex:
+        raise GraphQLError(f"Error testing Vercel credentials: {str(ex)}")
```

Comment:
This doesn't seem to be required, lets remove this resolver and the related import in `schema.py`

Reply:
Yeah, good call. Removed it. Regenerated graphql types.